src/module_sf_noahmplsm.F

```diff
@@ -4052,14 +4052,14 @@ SUBROUTINE VEGE_FLUX(parameters,NSNOW   ,NSOIL   ,ISNOW   ,VEGTYP  ,VEG     , &
 
         B   = SAV-IRC-SHC-EVC-TR+PAHV                          !additional w/m2
 !        A   = FVEG*(4.0*CIR*TV**3 + CSH + (CEV+CTR)*DESTV) !volumetric heat capacity
```

Comment:
maybe take this as a chance to remove this commented line, but maybe it's not necessary in this legacy code